lib/tasks/import.rake

```
@@ -280,28 +280,40 @@ namespace :import do
     end
   end
 
+  desc 'Schedule Re-consent Event for a particular date for all followed Participants'
+  task :schedule_reconsent => [:psc_setup, :environment, :set_whodunnit, :find_followed_participants_for_psc] do
+    date = Date.new(2013, 1, 1)
+    expected_followed_participants_for_psc.each do |pt|
+      if pt.date_available_for_informed_consent_event?(date)
+        resp = Event.schedule_reconsent(psc, pt, date)
+        msg = resp.success? ? "Re-consent event scheduled for #{pt.p_id}.\n" : "!!! Could not schedule informed consent for #{p.p_id} !!!\n"
+        $stderr.print(msg)
+        Rails.logger.info(msg)
+      end
```

This task is going to silently do nothing if the participant already has an informed consent on the selected date. Is that the right thing to do?